### PR TITLE
Chore/version bump

### PR DESCRIPTION
**Pull request by ammarkarachi**
#### Checklist

- [ ] PR description included
- [ ] `yarn test` passes
- [ ] Tests are [changed or added](https://github.com/aws-amplify/amplify-cli/blob/master/CONTRIBUTING.md#tests)
- [ ] Relevant documentation is changed or added (and PR referenced)
- [ ] New AWS SDK calls or CloudFormation actions have been added to relevant test and service IAM policies

By submitting this pull request, I confirm that my contribution is made under the terms of the Apache 2.0 license.

### CODE REVIEW

File: packages/amplify-cli/package.json [hunk 1]
```diff
@@ -33,43 +33,43 @@
...
-    "amplify-category-hosting": "3.0.0",
-    "amplify-category-interactions": "3.0.0",
+    "amplify-app": "4.2.0",
+    "amplify-category-analytics": "3.2.0",
+    "amplify-category-api": "3.2.0",
+    "amplify-category-function": "3.2.0",
+    "amplify-category-geo": "2.2.0",
+    "amplify-category-hosting": "3.2.0",
+    "amplify-category-interactions": "3.2.0",
     "amplify-category-notifications": "2.19.4",
```
Comment (kaustavghosh06) [inline packages/amplify-cli/package.json:47]: Why didn't this get bumped up?

File: packages/amplify-cli/package.json [hunk 2]
```diff
@@ -33,43 +33,43 @@
...
+    "amplify-category-predictions": "3.2.0",
+    "amplify-category-xr": "3.2.0",
+    "amplify-cli-core": "2.2.0",
     "amplify-cli-logger": "1.1.0",
     "amplify-codegen": "^2.23.1",
-    "amplify-console-hosting": "2.0.0",
-    "amplify-container-hosting": "2.0.0",
+    "amplify-console-hosting": "2.2.0",
+    "amplify-container-hosting": "2.2.0",
     "amplify-dotnet-function-runtime-provider": "1.6.3",
```
Comment (kaustavghosh06) [inline packages/amplify-cli/package.json:55]: Why didn't this get bumped up?

File: packages/amplify-cli/package.json [hunk 3]
```diff
@@ -33,43 +33,43 @@
...
-    "amplify-frontend-ios": "3.0.0",
-    "amplify-frontend-javascript": "3.0.0",
-    "amplify-go-function-runtime-provider": "2.0.0",
+    "amplify-dotnet-function-template-provider": "2.2.0",
+    "amplify-frontend-android": "3.2.0",
+    "amplify-frontend-flutter": "1.2.0",
+    "amplify-frontend-ios": "3.2.0",
+    "amplify-frontend-javascript": "3.2.0",
+    "amplify-go-function-runtime-provider": "2.2.0",
     "amplify-go-function-template-provider": "1.3.10",
```
Comment (kaustavghosh06) [inline packages/amplify-cli/package.json:62]: Why didn't this get bumped up?

File: packages/amplify-cli/package.json [hunk 4]
```diff
@@ -33,43 +33,43 @@
...
-    "amplify-java-function-runtime-provider": "2.0.0",
+    "amplify-java-function-runtime-provider": "2.2.0",
     "amplify-java-function-template-provider": "1.5.9",
```
Comment (kaustavghosh06) [inline packages/amplify-cli/package.json:64]: Why didn't this get bumped up?

File: packages/amplify-cli/package.json [hunk 5]
```diff
@@ -33,43 +33,43 @@
...
-    "amplify-nodejs-function-template-provider": "2.0.0",
-    "amplify-prompts": "1.3.0",
-    "amplify-provider-awscloudformation": "5.0.0",
-    "amplify-python-function-runtime-provider": "2.0.0",
+    "amplify-nodejs-function-runtime-provider": "2.2.0",
+    "amplify-nodejs-function-template-provider": "2.2.0",
+    "amplify-prompts": "1.5.0",
+    "amplify-provider-awscloudformation": "5.2.0",
+    "amplify-python-function-runtime-provider": "2.2.0",
     "amplify-python-function-template-provider": "1.3.12",
```
Comment (kaustavghosh06) [inline packages/amplify-cli/package.json:70]: Why didn't this get bumped up?

File: packages/amplify-cli/package.json [hunk 6]
```diff
@@ -33,43 +33,43 @@
...
+    "amplify-category-hosting": "3.2.0",
+    "amplify-category-interactions": "3.2.0",
     "amplify-category-notifications": "2.19.4",
-    "amplify-category-predictions": "3.0.0",
-    "amplify-category-xr": "3.0.0",
-    "amplify-cli-core": "2.0.0",
+    "amplify-category-predictions": "3.2.0",
+    "amplify-category-xr": "3.2.0",
+    "amplify-cli-core": "2.2.0",
     "amplify-cli-logger": "1.1.0",
```
Comment (kaustavghosh06) [inline packages/amplify-cli/package.json:51]: Why didn't this get bumped up?

File: packages/amplify-cli/package.json [hunk 7]
```diff
@@ -33,43 +33,43 @@
...
     "amplify-codegen": "^2.23.1",
```
Comment (kaustavghosh06) [inline packages/amplify-cli/package.json:52]: Is this package in our repo?

File: packages/amplify-dotnet-function-template-provider/package.json [hunk 8]
```diff
@@ -22,9 +22,9 @@
     "clean": "rimraf lib tsconfig.tsbuildinfo"
   },
   "dependencies": {
-    "amplify-cli-core": "2.0.0",
+    "amplify-cli-core": "2.2.0",
     "amplify-function-plugin-interface": "1.9.1",
```
Comment (kaustavghosh06) [inline packages/amplify-dotnet-function-template-provider/package.json:26]: Why didn't this get bumped up?

File: packages/amplify-go-function-runtime-provider/package.json [hunk 9]
```diff
@@ -21,7 +21,7 @@
     "clean": "rimraf lib tsconfig.tsbuildinfo resources/localinvoke/go.sum resources/localinvoke/main resources/localinvoke/main.exe"
   },
   "dependencies": {
-    "amplify-cli-core": "2.0.0",
+    "amplify-cli-core": "2.2.0",
     "amplify-function-plugin-interface": "1.9.1",
```
Comment (kaustavghosh06) [inline packages/amplify-go-function-runtime-provider/package.json:25]: Why didn't this get bumped up?

File: packages/amplify-java-function-runtime-provider/package.json [hunk 10]
```diff
@@ -21,7 +21,7 @@
     "clean": "rimraf lib tsconfig.tsbuildinfo"
   },
   "dependencies": {
-    "amplify-cli-core": "2.0.0",
+    "amplify-cli-core": "2.2.0",
     "amplify-function-plugin-interface": "1.9.1",
```
Comment (kaustavghosh06) [inline packages/amplify-java-function-runtime-provider/package.json:25]: Why didn't this get bumped up?

File: packages/amplify-nodejs-function-runtime-provider/package.json [hunk 11]
```diff
@@ -23,7 +23,7 @@
     "test": "jest"
   },
   "dependencies": {
-    "amplify-cli-core": "2.0.0",
+    "amplify-cli-core": "2.2.0",
     "amplify-function-plugin-interface": "1.9.1",
```
Comment (kaustavghosh06) [inline packages/amplify-nodejs-function-runtime-provider/package.json:27]: Why didn't this get bumped up?

File: packages/amplify-nodejs-function-template-provider/package.json [hunk 12]
```diff
@@ -21,9 +21,9 @@
     "clean": "rimraf lib tsconfig.tsbuildinfo"
   },
   "dependencies": {
-    "amplify-cli-core": "2.0.0",
+    "amplify-cli-core": "2.2.0",
     "amplify-function-plugin-interface": "1.9.1",
```
Comment (kaustavghosh06) [inline packages/amplify-nodejs-function-template-provider/package.json:25]: Why didn't this get bumped up?

File: packages/amplify-provider-awscloudformation/package.json [hunk 13]
```diff
@@ -76,10 +76,10 @@
     "@aws-cdk/region-info": "~1.124.0",
     "@octokit/rest": "^18.0.9",
     "ajv": "^6.12.3",
-    "amplify-cli-core": "2.0.0",
+    "amplify-cli-core": "2.2.0",
     "amplify-cli-logger": "1.1.0",
```
Comment (kaustavghosh06) [inline packages/amplify-provider-awscloudformation/package.json:80]: Why didn't this get bumped up?

File: packages/amplify-provider-awscloudformation/package.json [hunk 14]
```diff
@@ -76,10 +76,10 @@
...
     "amplify-codegen": "^2.23.1",
```
Comment (kaustavghosh06) [inline packages/amplify-provider-awscloudformation/package.json:81]: is this in our repo?

File: packages/amplify-python-function-runtime-provider/package.json [hunk 15]
```diff
@@ -21,7 +21,7 @@
     "clean": "rimraf lib tsconfig.tsbuildinfo"
   },
   "dependencies": {
-    "amplify-cli-core": "2.0.0",
+    "amplify-cli-core": "2.2.0",
     "amplify-function-plugin-interface": "1.9.1",
```
Comment (kaustavghosh06) [inline packages/amplify-python-function-runtime-provider/package.json:25]: Why didn't this get bumped up?

File: packages/amplify-cli/package.json (hunk 2 above)
Comment (ammarkarachi) [reply packages/amplify-cli/package.json:55]: It's bumped 2.0.0 -> 2.2.0

**Review by kaustavghosh06 — APPROVED** [review]

**Pull request merged**